policyengine_us/parameters/gov/states/mt/tax/income/main/capital_gains/rates/surviving_spouse.yaml

```
@@ -0,0 +1,21 @@
+description: Montana taxes the net long-term capital gains by the following marginal rates for widow filers, according to the taxable income.
```

Also shows up in a few other places in the pr
```suggestion
description: Montana taxes the net long-term capital gains by the following marginal rates for surviving spouse filers, according to the taxable income.
```